src/extension.ts

```diff
@@ -1052,6 +1053,15 @@ export async function activate(context: vscode.ExtensionContext) {
         await vscode.window.showWarningMessage(localize('uninstall.old.cmaketools', 'Please uninstall any older versions of the CMake Tools extension. It is now published by Microsoft starting with version 1.2.0.'));
     }
 
+    // Rescan if the kits version (saved in the extension state) is not the same as the extension version,
+    // to avoid eventual breaking changes in the kits definitions.
+    const kitsVersion = context.workspaceState.get<string>('kitsVersion');
+    const extVersion = vscode.extensions.getExtension('ms-vscode.cmake-tools')?.packageJSON.version;
```

Comment:
This can just be a const number that we increment whenever the kits version changes.

Reply:
Ok.